controllers/controllers.go

```diff
@@ -110,8 +118,23 @@ func (r *instanceReconciler) ensureInstanceIsUpToDate(instanceInfo *instance.Inf
 	return nc.Configure()
 }
 
+// getNodeSummary queries the cAdvisor metrics endpoint for the given Node
+func (r *instanceReconciler) getNodeSummary(nodeName string) (stats.Summary, error) {
```

Comment:
I suggest just returning `stats.Summary.Pods` given that is all we need and rename the function appropriately. Example: `getPodStatsForNode()`.